package.json

```
@@ -2,23 +2,27 @@
   "name": "tika-text-extract",
```

```suggestion
  "name": "@shelf/tika-text-extract",
```

It was previously under my personal user. Let's make it on behalf of Shelf